### ISSUE TITLE
Error running filebeat setup illegal state exception for ML

### ISSUE THREAD

**Issue by liza-mae** [elastic/beats#9574]
Version 7.0 Alpha2 

Loaded dashboards
Exiting: 1 error: Error setting up ML for nginx: 5 errors: [illegal_state_exception] parser for [_doc] did not end on END_OBJECT; [illegal_state_exception] parser for [_doc] did not end on END_OBJECT; [illegal_state_exception] parser for [_doc] did not end on END_OBJECT; [illegal_state_exception] parser for [_doc] did not end on END_OBJECT; [illegal_state_exception] parser for [_doc] did not end on END_OBJECT

**Comment by liza-mae** [comment]
I am running with elastic user

**Comment by ruflin** [comment]
@liza-mae I assume alpha2 of Beats and Elasticsearch? @urso Looks like a `_doc` issue?

**Comment by liza-mae** [comment]
@ruflin yes alpha2 for both Beats and Elasticsearch

**Comment by ruflin** [comment]
Could you share the exact command you were using? I tried this in Beats `master` and running `./filebeat setup -E setup.dashboards.directory=_meta/kibana.generated/`. The exact Elasticsearch build I used is:

```
number: "7.0.0",
build_hash: "a181a25",
build_date: "2018-12-17T15:07:49.156295Z",
```

**Comment by ruflin** [comment]
I now tested this again with the release version of alpha2. It seems also with this version I can't reproduce it. There were some hiccups with `_doc` so it could be that this got resolved before we released 7.0.0-alpha2.

@liza-mae Could you try this again with the released 7.0.0-alpha2 version to see if it is also fixed on your end. For the flags: You don't need them, that was only needed because I run it from master in the Beats repo.

**Comment by kvch** [comment]
Filebeat no longer uploads ML jobs in the command `setup`.
